### PR TITLE
EUI-4100 Task check at start multiple tasks

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4100

### Change description ###
Task triggered from Tasks tab

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by kuda1992 — APPROVED** [review]
Looks good to me 😊

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

File: src/cases/components/case-task/case-task.component.ts [hunk 1]
```diff
@@ -62,6 +62,15 @@ export class CaseTaskComponent implements OnInit {
     if (!task.description) {
       return '';
     }
+    // Append task id as a querystring to the task description url
+    // The url format will be like the following markdown
+    // 'Click link to proceed to next step [test link next step](/cases/case-details/1547652071308205/trigger/editAppealAfterSubmit)'
+    const taskDescription = task.description;
```
Comment (udaydenduluri33) [inline src/cases/components/case-task/case-task.component.ts:68]: Can you move this logic to a util function and write some unit tests just for that?

Comment (udaydenduluri33) [reply src/cases/components/case-task/case-task.component.ts:68]: Please create a new src/cases/components/case-task/src/cases/components/case-task/case-task.util.ts

Comment (johnbenjamin-hmcts) [reply src/cases/components/case-task/case-task.component.ts]: Thank you @udaydenduluri33, I have now implemented the code as you suggested.

**Review by udaydenduluri33 — APPROVED** [review]
@johnbenjamin-hmcts thanks for making the changes

File: src/cases/components/case-task/case-task.util.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  // The url format will be like the following markdown:
+  // 'Click link to proceed to next step [test link next step](/cases/case-details/1547652071308205/trigger/editAppealAfterSubmit)'
+
+  // For Task Event Start and Completion, when user intiates an event from the tasks tab
+  // we need a way to identify the task chosen by the user for further processing
+  // The Next steps link is configured and provided by the backend services
+  // So, instead of modifying the url, we decided that the safest approach is to
+  // append task id as a querystring.
+
+  const taskDescription = task.description;
```
Comment (udaydenduluri33) [inline src/cases/components/case-task/case-task.util.ts:13]: do we need to check if task is null
`if(task && task.description) {...`

Comment (johnbenjamin-hmcts) [reply src/cases/components/case-task/case-task.util.ts]: Thank you @udaydenduluri33, I have applied the changes.

**Pull request merged**